drivers/input/Kconfig.pinnacle

```diff
@@ -0,0 +1,32 @@
+# Cirque Pinnacle 1CA027 touch controller based device configuration options
+#
+# Copyright (c) 2024 Ilia Kharin
+# SPDX-License-Identifier: Apache-2.0
+
+config INPUT_PINNACLE
+	bool "Cirque Pinnacle 1CA027 Touch Controller Based Device"
+	default y
+	depends on DT_HAS_CIRQUE_PINNACLE_ENABLED
+	select GPIO
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_CIRQUE_PINNACLE),i2c)
+	select SPI if $(dt_compat_on_bus,$(DT_COMPAT_CIRQUE_PINNACLE),spi)
+	help
+	  Enable driver for Cirque Pinnacle 1CA027 tracked input device.
+
+if INPUT_PINNACLE
+
+choice INPUT_PINNACLE_DATA_MODE
+	prompt "Data mode"
+	default INPUT_PINNACLE_DATA_MODE_ABSOLUTE
+	help
+	  Specify the type of the data mode.
+
+config INPUT_PINNACLE_DATA_MODE_ABSOLUTE
+	bool "Absolute data mode"
+
+config INPUT_PINNACLE_DATA_MODE_RELATIVE
+	bool "Relative data mode"
```

Comment:
these two could probably use a more verbose description, may also be worth thinking on whether they should be a devictree option, in case someone wants to have two of these in different mode, does it make sense?

Reply:
Moved to a DT enum option.